README.md

```diff
@@ -57,8 +57,8 @@ By default, ESLint will attempt to fix whatever errors possible; the rest should
 #### Links
 Resource | URL
 ------------ | -------------
- API | https://api.hackpsu.org/v1
- API Documentation | https://api.hackpsu.org/v1/doc
+ API | https://api.hackpsu.org/v2
```

Comment:
So here, the v2 is not necessary (the link goes no where) so it should be just https://api.hackpsu.org/ (which displays a json if you go to it)

Reply:
I'll fix it in my commit